incubator/kafka/templates/statefulset.yaml

```diff
@@ -173,6 +173,10 @@ spec:
         - name: KAFKA_JMX_PORT
           value: "{{ .Values.jmx.port }}"
         {{- end }}
+        {{- range $key, $value := .Values.env }}
```

Comment:
This isnt specified in the `values.yaml` -- is this something that could be passing through `configurationOverrides`?

Reply:
I believe that `env` might be more useful here if someone is considering using `envFrom` from either configMap or  Secret which might be the case as this PR has secrets added as well.  `configurationOverrides` might still be used here.

Reply:
This may be a secret in the following format:
```
  env:
  - name: PASSWORD
     valueFrom:
       secretKeyRef:
         name: supersecret
         key: password
```

Reply:
Oh I see where its broke now.

@rustycl0ck any way of supporting doing this.  The value is explicit and there is no support for `valueFrom`